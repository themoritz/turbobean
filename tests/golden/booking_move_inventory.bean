2020-01-01 open Assets:Stocks     AAPL "FIFO"
2020-01-01 open Assets:MoreStocks AAPL "FIFO"
2020-01-01 open Assets:Cash

2020-01-02 * "Buy"
  Assets:Stocks   10 AAPL @ 10 USD
  Assets:Cash   -100 USD

2020-01-03 txn "Move"
  Assets:Stocks      -10 AAPL {10 USD, 2020-01-02} @ 1 USD
  Assets:MoreStocks   10 AAPL {10 USD, 2020-01-02} @ 1 USD
