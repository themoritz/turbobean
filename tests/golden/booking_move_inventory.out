Assets         10 AAPL @ 10 USD {2020-01-02}
               -100 USD
├ Stocks
├ MoreStocks   10 AAPL @ 10 USD {2020-01-02}
└ Cash         -100 USD
